### DISCUSSION TITLE
Connect serial port to TTGO board

### DISCUSSION THREAD

**Discussion by theshinyknight** [fdivitto/FabGL#381] [Q&A]
Is there a way to connect a serial port to the TTGO board running FabGL? Would like to connect a Tandy computer to it and use the TTGO as serial server.

I see that the board has 8 pins; would these be fit for a serial connection?

**Comment by FernandoRoxo** [comment]
There is a video in YouTube that connects a SBC Z80 to a TTGO board via serial.   Hope this helps:


[https://www.youtube.com/watch?v=zhszathMpg](https://www.youtube.com/watch?v=zhszathMpgY)
